databooks/data_models/cell.py

```diff
@@ -250,8 +253,19 @@ def __rich_console__(
     def values(
         self,
     ) -> List[CellOutputType]:
-        """Alias `__root__` with outputs for easy referencing."""
-        return self.__root__
+        """Alias `root` with outputs for easy referencing."""
+        return self.root
+
+    # Before CellOutputs inherited from RootModel
+    # eq worked well (suppose covered by DatabooksBase,
+    # which inherits from Pydantic's BaseModel)
```

Comment:
You mean that `RootModel`s  `__eq__` does not work well now? What's the issue with it? It doesn't compare `type` and `model_config`?

Reply:
Yes - it appears it does not.

To be fair, I am not exactly certain right now, it's been a few days, good that you asked... I have an idea to go through a debugger once more and see which `__eq__` method it eventually enters up the inheritance chain.

Reply:
Ok so:

1. `RootModel`'s `__eq__`` works fine.

Reply:
Sounds great! if it's still needed, we should maybe just be a bit more descriptive (i.e.: what's the issue exactly)